<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IoCardTest</class>
 <widget class="QWidget" name="IoCardTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IoCardTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="pBtn_Init">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="pushButton_CloseDev">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string> 关闭</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="pushButton_CanopyGreen">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>雨棚绿灯</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="pushButton_PassGreen">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>通行绿灯</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="pushButton_BarriarUp">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>抬杆</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="pushButton_StartAlarm">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>开始报警</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="pushButton_CanopyRed">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>雨棚红灯</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="pushButton_PassRed">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>通行红灯</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="pushButton_BarriarDown">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>落杆</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="pushButton_EndAlarm">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>取消报警</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBox_front">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>存在线圈</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="checkBox_up">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>抬杆到位</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBox_back">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>通过线圈</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="checkBox_down">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>落杆到位</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>显示信息</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
